tests/nsrt_learning/strips_learning/test_gen_to_spec_learner.py

```diff
@@ -111,6 +117,134 @@ def test_backchaining_strips_learner():
     assert str(pnads[0]) == repr(pnads[0]) == expected_str
 
 
+def test_backchaining_strips_learner_order_dependence():
+    """Test that the BackchainingSTRIPSLearner is invariant to order of
+    traversal through trajectories."""
+    # Setup up the types and predicates.
```

Comment:
Set up*

(also, this appears in behavior_options.py, so if you want, you can update "Setup" -> "Set up" there too)